### PR TITLE
[lint] applying go-lint suggestions

### PR DESCRIPTION
**Pull request by benjaminch**
Applying go-lint suggestions

### CODE REVIEW

**Comment by benjaminch** [comment]
Thanks @mansinahar :)

File: pbs_light_test.go [hunk 1]
```diff
@@ -28,11 +28,10 @@ func forceEnv(t *testing.T, key string, val string) func() {
 				t.Fatalf("Error unsetting evnvironment %s", key)
```
Comment (guscarreon) [inline pbs_light_test.go]: I realize you didn't misspell this word but, could you correct it? Unless it is a hassle to go through another round of reviews. If so, we can live with it.

Comment (benjaminch) [reply pbs_light_test.go]: No worries I’ll handle it !

Comment (benjaminch) [reply pbs_light_test.go]: Here you go: #996

**Pull request merged**